src/test/vscodeUiTest/suite/projectGenerationTest.ts

```diff
@@ -31,13 +31,19 @@ use(require('chai-fs'));
  */
 describe('Project generation tests', function() {
```

Comment:
maybe try process.env.VSCODE_QUARKUS_API_URL=https://stage.code.quarkus.io/api directly here. 
Dunno if it'll work

Reply:
fixed